homepairs/homepairsUI/src/Elements/Tiles/styles.web.tsx

```diff
@@ -10,6 +10,7 @@ const textStyleBase = {
 
 const styles = StyleSheet.create({
     imageTileContainer: {
+        flex: 1,
```

Comment:
I'm going to make sure this doesn't affect any of Cesar's or my components.

Reply:
Adding all the flexes were screwing with my tiles on web. I'm going to go ahead and fix this.

Reply:
sounds good. I think alignItems fixed mine but I've been staring at a computer for 10 hours so I took the better safe than sorry approach and added flex